blob-polyfill.ts

```diff
@@ -0,0 +1,95 @@
+import { getArrayBufferForBlob } from "react-native-blob-jsi-helper";
+import { ReadableStream } from "web-streams-polyfill";
+
+const decoder = new TextDecoder();
+
+/**
+ * React Native's Blob implementation has a constructor that currently only supports
+ * constructing from parts that are of type: Array<Blob | String>.
```

Comment:
React Native's Blob can be found at `./node_modules/react-native/Libraries/Blob/Blob.js`